docs/ad/api.md

```diff
@@ -238,6 +238,56 @@ POST _opendistro/_anomaly_detection/detectors
 }
 ```
 
+To create a historical detector:
+
+#### Request
+
+```json
+POST _opendistro/_anomaly_detection/detectors
+{
+  "name": "test1",
+  "description": "test historical detector",
+  "time_field": "timestamp",
+  "indices": [
+    "nab_art_daily_jumpsdown"
```

Comment:
Minor, how about we use the same index name ? I see we used "order*", "host-cloudwatch" for realtime detector and HC detector, now we use "nab_art_daily_jumpsdown"